shpc/client/install.py

```diff
@@ -36,5 +35,4 @@ def main(args, parser, extra, subparser):
             cli.settings.default_view,
             args.install_recipe,
             force=args.force,
-            container_image=args.container_image,
```

Comment:
okay, and this will work because we are symlinking the module file, which has the path to the container, wherever it may be!